consensus/replay.go

```diff
@@ -456,9 +462,26 @@ func (h *Handshaker) replayBlock(state sm.State, height int64, proxyApp proxy.Ap
 	return state, nil
 }
 
-func checkAppHash(state sm.State, appHash []byte) error {
-	if !bytes.Equal(state.AppHash, appHash) {
-		panic(fmt.Errorf("Tendermint state.AppHash does not match AppHash after replay. Got %X, expected %X", appHash, state.AppHash).Error())
+func checkAppHashAgainstBlock(appHash []byte, block *types.Block) error {
```

Comment:
Why does the methiod return an error? The only thing it does, is panic in an error case and return nil if everything seems right.

Reply:
Good catch. No need to return an error